src/test/kotlin/org/rust/ide/inspections/import/AutoImportFixTestBase.kt

```diff
@@ -113,3 +96,23 @@ abstract class AutoImportFixTestBase : RsInspectionsTestBase(RsUnresolvedReferen
         myFixture.checkResult(text)
     }
 }
+
+fun checkAutoImportWithMultipleChoice(expectedElements: List<String>, choice: String?, action: () -> Unit) {
+    var areElementEquals: Boolean? = null
+
+    withMockImportItemUi(object : ImportItemUi {
+        override fun chooseItem(items: List<ImportCandidate>, callback: (ImportCandidate) -> Unit) {
+            val actualItems = items.map { it.info.usePath }
+            areElementEquals = expectedElements == actualItems
+            BasePlatformTestCase.assertEquals(expectedElements, actualItems)  // exception here does not fail the test
```

Comment:
```suggestion
           assertEquals(expectedElements, actualItems)  // exception here does not fail the test
```

Don't forget to remove unnecessary import after this change